packages/rancher-istio/charts/1.6.400/templates/istio-install-job.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: istioctl-installer
+  namespace: istio-system
+  annotations:
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+spec:
+  backoffLimit: 1
+  ttlSecondsAfterFinished: 100
```

Comment:
I don't think this actually works because its feature gated: https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/